js/main.js

```diff
@@ -31,27 +31,27 @@ $centuryButtons.forEach(item => {
   });
 });
 
-if (data.view === 'page2') {
-  $body.className = 'page2-bg';
+if (data.view === 'art-periods-row') {
+  $body.className = 'art-period-bg';
   $divPage1.className = 'hidden row';
-  $divPage2.className = 'page2-row';
+  $divPage2.className = 'art-periods-row';
 }
 
 $nextButton.addEventListener('click', changePage2);
 
 function changePage2(event) {
```

Comment:
This function name doesn't describe the action that is taking place. Which page are you changing to?